Documentation/MinimalExample/Index.rst

```diff
@@ -3,98 +3,184 @@
 
 .. _minimal-design:
 
-===============
-Minimal example
-===============
+====================
+Minimal site package
+====================
 
-We want to create a site package that outputs a single web page with
-minimal effort. This site package can be used to simply test system output or as
-an example of the fewest possible steps to create a working site package.
+A site package is a custom TYPO3 extension which contains configuration,
+templates, assets etc that are used for the site it belongs to.
 
-To start, in the TYPO3 backend, create a standard page named
-:guilabel:`Minimal example` just under (inside) the page tree TYPO3 logo
-container. Create a new TypoScript template record on this page.
-Give the TypoScript template a title, and make it a root level template,
-but do not include any static templates.
+So we first must create a minimal extension.
+
+.. _minimal-extension:
+
+Create a minimal TYPO3 extension using b13/make
+===============================================
+
+:composer:`b13/make` is a convenient TYPO3 extension which you can use during
```

Comment:
```suggestion
:composer:`b13/make` is an easy TYPO3 extension which you can use during
```

Reply:
@sarahmccarthy123 We once where told in a workshop to avoid the word "easy" in the docs as that can frustrate users that ar new and don't find it easy. Can you think of another word I can use here?

Reply:
Would "simple" work? I just think "convenient" is a bit meaningless in a software context

Reply:
Simple and easy were both mentioned as words to be avoided in documentation

How about "... is a useful TYPO3 extension that assists you during ..."